shared/initialState.js

```diff
@@ -140,6 +140,7 @@ const line = {
   id: 1,
   bookId: 1,
   color: '#6cace4',
+  backgroundColor: '#6cace419',
```

Comment:
oh, i see now what you meant about this. Why does this have to be a separate attribute?

Reply:
> I know this is going to cause a lot of conflicts with my changes … yippeee
> 
> I think we should hold off merging this until next week (we're planning to release this week). Is that going to make any future changes for you more difficult?

Nope.  I can totally hold off and keep rebasing my latest branches.  No problem :+1:

Reply:
It gets added programmatically when we render the line, so I'm not sure how else to specify an exact colour.
So in this PR I only made it a less opaque version of the original colour, but I'm not sure that that's perfect, style-wise.
If we settle on it just being lighter, then we can compute it from color using something like `tinycolor` https://www.npmjs.com/package/tinycolor2.

I might be drawing blanks here though.  Can you think of something better?